src/test/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -259,4 +282,102 @@ public void testLock() throws Exception {
 
     this.subject.lock();
   }
+
+  @Test
+  public void testBatch() throws Exception {
+    JsonObject json = Json.createObjectBuilder()
+      .add("headers", Json.createObjectBuilder()
+          .add("sc-unit-system", "metric"))
+      .add("requests", Json.createArrayBuilder()
+          .add(Json.createObjectBuilder()
+            .add("path", "/odometer")
+            .build()
+          )
+          .add(Json.createObjectBuilder()
+            .add("path", "/fuel")
+            .build()
+          )
+          .build()
+      )
+      .build();
+    RequestBody body = RequestBody.create(Vehicle.JSON, json.toString());
+    String expectedJson = "[{\"headers\":{\"unitSystem\":\"metric\"},\"path\":\"/odometer\",\"code\":200,\"body\":{\"distance\":32768}}]";
```

Comment:
is there another way to write the expected JSON that it's easier to maintain? (i.e. using a builder to construct this response).

Reply:
yes, moved to seperate file!